### PR TITLE
Make social links and invite to slack form.

### PR DESCRIPTION
**Pull request by geowa4**
Created a slack invite form and hooked it up to actually invite people.
Additionally, cleaned up styles and tweaked the design.

Implements part of #39 since it does not show how many have currently
signed up or are online at the moment.

### CODE REVIEW

File: app/lib/rocdev/chat/chat_inviter.ex [hunk 1]
```diff
@@ -0,0 +1,47 @@
+defmodule Rocdev.ChatInviter do
+  @moduledoc """
+  Provides access to Slack Invitation API.
+  """
+  use HTTPoison.Base
```
Comment (geowa4) [inline app/lib/rocdev/chat/chat_inviter.ex:5]: From my Googling, this seems to be the way to go.

File: app/mix.lock [hunk 2]
```diff
@@ -28,4 +28,5 @@
   "ranch": {:hex, :ranch, "1.3.2", "e4965a144dc9fbe70e5c077c65e73c57165416a901bd02ea899cfd95aa890986", [], [], "hexpm"},
   "sparkpost": {:hex, :sparkpost, "0.5.1", "59d85a140fa4d5d40ee8ec970cf38fe5f7661a940595b16d565439c88e39fd52", [], [{:httpoison, "~> 0.9", [hex: :httpoison, repo: "hexpm", optional: false]}, {:poison, "~> 2.0 or ~> 3.0", [hex: :poison, repo: "hexpm", optional: false]}], "hexpm"},
   "ssl_verify_fun": {:hex, :ssl_verify_fun, "1.1.1", "28a4d65b7f59893bc2c7de786dec1e1555bd742d336043fe644ae956c3497fbe", [], [], "hexpm"},
+  "tesla": {:hex, :tesla, "0.9.0", "db5c86895b63e698bbc61d62b4fd32735112de80ed82dcceef3660679fed9324", [], [{:exjsx, ">= 0.1.0", [hex: :exjsx, repo: "hexpm", optional: true]}, {:fuse, "~> 2.4", [hex: :fuse, repo: "hexpm", optional: true]}, {:hackney, "~> 1.6", [hex: :hackney, repo: "hexpm", optional: true]}, {:ibrowse, "~> 4.2", [hex: :ibrowse, repo: "hexpm", optional: true]}, {:mime, "~> 1.0", [hex: :mime, repo: "hexpm", optional: false]}, {:poison, ">= 1.0.0", [hex: :poison, repo: "hexpm", optional: true]}], "hexpm"},
```
Comment (geowa4) [inline app/mix.lock]: Tesla was removed from mix.exs so why is it in mix.lock?

Comment (dantswain) [reply app/mix.lock]: It does that.  I'm not entirely sure why.  `mix deps.unlock tesla` will remove it.

File: app/config/config.exs [hunk 3]
```diff
@@ -23,7 +25,7 @@ config :logger, :console,
   metadata: [:request_id]
 
 config :sparkpost,
-  api_key: "TODO SUPPLY KEY",
+  api_key: System.get_env("SPARKPOST_API_KEY") || "${SPARKPOST_API_KEY}",
```
Comment (dantswain) [inline app/config/config.exs]: I don't think the`"${....}"` will do any magic.  If you want to be able to configure them at runtime, ping me and I'll show you how.

Comment (geowa4) [reply app/config/config.exs]: Ugh, I forgot to remove that.

**Pull request merged**